### DISCUSSION TITLE
is there any difference between x10-plots on 2 different PCs vs x20-plots on 1 PC ?

### DISCUSSION THREAD

**Discussion by ch3ll0v3k** [Chia-Network/chia-blockchain#2855] [General]
Hi. i was wondering if there is any difference in chance to get coins ? 

  1. there are 2 PCs with each 10 (different on each pc) plot in farm 
  Or
  2. or just 1 PC with 20 different plots in one farm.

**Comment by dorkmo** [comment]
both the same. every plot connected to a node has the same chance.

**Reply by ch3ll0v3k** [reply]
thx
